internal/backends/python/python.go

```diff
@@ -824,7 +857,10 @@ func makePythonUvBackend() api.LanguageBackend {
 				}
 
 				cmd = append(cmd, pep440Join(name, spec))
+
+				addExtraIndexes(string(name))
```

Comment:
Unfortunately we need to do the full `listUvSpecfile()` dance in `Add` as well, since we could be `uv add`-ing into a project that has unsync'd dependencies already in the project file

Reply:
Ah, good point!